src/components/oscal-utils/OSCALComponentResolver.js

```diff
@@ -9,16 +9,35 @@ export default function OSCALComponentResolveSources(
   // Fixing linting error here would take significant change to codebase given how we use props.
   /* eslint no-param-reassign: "error" */
   componentDefinition.resolvedControls = [];
+
+  let pendingProcesses = componentDefinition.components
```

Comment:
```suggestion
  const pendingProcesses = componentDefinition.components
```

will this work?

Reply:
No, because it needs to be decremented later.

Reply:
But maybe `var` instead? not sure what's preferred between `let` and `var` but this is function-scoped which works with `var`.

Reply:
We want `let` over `var` in basically ever scenario. But we like `const` the most but I forgot this is mutable. So we need `let`. Thanks for explaining!